custom_dc/env.list.sample

```diff
@@ -12,9 +12,10 @@ MAPS_API_KEY=
 # Local paths must be absolute paths.
 INPUT_DIR=
 # Output directory in Google Cloud Storage (GCS) or local drive for generated csv files.
-# GCS paths must be of the form gs://<BUCKET>/<FOLDER>.
+# GCS paths must be of the form gs://<BUCKET>/<FOLDER>. If you specify a GCS path, be 
```

Comment:
nit: Perhaps start the additional sentence on a new line.

Reply:
Will do.